scripts/publish.sh

```diff
@@ -21,9 +21,19 @@ git add dist --force
 git commit -m "Added files generated by Travis build"
 
 # Push to releases branch
+EXISTING=`git ls-remote --heads https://github.com/"$TRAVIS_REPO_SLUG".git "$TRAVIS_BRANCH"-dist`
+
 if [ -z "$TRAVIS_TAG" ]
 then
-  git push upstream $TRAVIS_BRANCH:$TRAVIS_BRANCH-dist --force -v
+  if [ -n "$EXISTING" ]
+  then
+    git fetch upstream $TRAVIS_BRANCH-dist:$TRAVIS_BRANCH-dist
+    git checkout $TRAVIS_BRANCH-dist
+    git merge -Xtheirs $TRAVIS_BRANCH --no-edit --ff
+    git push upstream $TRAVIS_BRANCH-dist --force -v
```

Comment:
@dlabrecq this doesn't appear to work. See my example run [here](https://travis-ci.org/priley86/angular-patternfly/builds/152482219). 

```
git push upstream master:master-dist -v
Pushing to https://1d0020e942c6eb33f9a70eb46935b329e79cf8f6@github.com/priley86/angular-patternfly.git
To https://1d0020e942c6eb33f9a70eb46935b329e79cf8f6@github.com/priley86/angular-patternfly.git
 ! [rejected]        master -> master-dist (fetch first)
error: failed to push some refs to 'https://1d0020e942c6eb33f9a70eb46935b329e79cf8f6@github.com/priley86/angular-patternfly.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Problem is without force, you will have missing commits (previous automated commits) that are currently in `master-dist` when attempting to push `master` to `master-dist`. 

Any other ideas? I believe the merge may be necessary...